tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "files": [
         "plugins/PlatformFSPlugin.ts",
         "plugins/WatchStateLoggerPlugin.ts",
+        "plugins/ns-replace-bootstrap.ts",
```

Comment:
mm, that's not really a webpack plugin. Can we move it to a `transformers` folder or something like that.